Tasks/VSMobileCenterUpload/utils.ts

```diff
@@ -0,0 +1,120 @@
+import path = require("path");
+import tl = require("vsts-task-lib/task");
+import fs = require("fs");
+import Q = require('q');
+import glob = require('glob');
+
+var Zip = require('jszip');
+
+export function checkAndFixFilePath(p, name) {
+    if (p) {
+        var workDir = tl.getVariable("System.DefaultWorkingDirectory");
+        var sourcesDir = tl.getVariable("Build.SourcesDirectory");
```

Comment:
Eric Sciple confirmed that `System.DefaultWorkingDirectory` always holds the same value as `Build.SourcesDirectory`, so we can use just the first one since it is present in both Build and RM scenarios.